modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -546,20 +550,19 @@ export default {
       this.isLoading = false;
       this.skipLoadObserver = false;
     },
-
-    async onContentChanged({ action, doc }) {
-      if (doc.type !== '@apostrophecms/image' || ![ 'archive', 'update' ].includes(action)) {
+    async onContentChanged({
+      action, doc, docTypes
+    }) {
+      const types = this.getContentChangedTypes(doc, docTypes);
```

Comment:
Moved the logic in a mixin, because it's the same for all managers listening `content-changed ` (it handles pages, draft submitted, single and batch operations).